packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -892,18 +892,8 @@ module('Integration | operator-mode', function (hooks) {
           .querySelector('[data-test-auto-save-indicator]')
           ?.textContent?.trim() == 'Saving…',
     );
-    await waitUntil(
-      () =>
-        document
-          .querySelector('[data-test-auto-save-indicator]')
-          ?.textContent?.trim() == 'Failed to save: Boom!',
-    );
-    setCardInOperatorModeState(`${testRealmURL}BoomPet/paper`);
-
-    await waitFor('[data-test-pet]');
-    // Card still runs (our error was designed to only fire during save)
-    // despite save error
-    assert.dom('[data-test-pet]').includesText('Paper Bad cat!');
```

Comment:
this is the test for which i recorded a video in the PR description. us used to be able to show a running card because in this particular scenario the live update was not actually working. now the live update is much more consistent thru error states.